haiku-apps/genio/genio-2.0.0.recipe

```diff
@@ -0,0 +1,61 @@
+SUMMARY="The Haiku IDE"
+DESCRIPTION="Genio is a native IDE for the Haiku operating system. \
+It has a rich editor supporting syntax highlight, auto completion and other advanced features. \
+It contains a set of templates to quickly develop and build your projects."
+HOMEPAGE="https://github.com/Genio-The-Haiku-IDE/Genio/releases"
+COPYRIGHT="2022-2023 The Genio Team"
+LICENSE="BSD (3-clause)
+	MIT"
+REVISION="1"
+srcGitRev="eae9cf7a34c099a9dfb579aa0a478bf5e7840836"
+SOURCE_URI="https://github.com/Genio-The-Haiku-IDE/Genio/archive/v2.tar.gz"
+CHECKSUM_SHA256="97e46dd5523702a4c7d91c68aee7806d4acab9a7b4b58fe4756429d00c101ffc"
+SOURCE_FILENAME="Genio-v2.tar.gz"
+SOURCE_DIR="Genio-2"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	genio${secondaryArchSuffix} = $portVersion
+	app:Genio = $portVersion
+	"
+REQUIRES="
+	haiku${secondaryArchSuffix}
+	gcc_syslibs${secondaryArchSuffix}_devel
+	llvm17${secondaryArchSuffix}_clang
+	lib:libgit2${secondaryArchSuffix}
+	lib:libyaml_cpp${secondaryArchSuffix}
```

Comment:
These brackets are not needed here for lib:lib* (same for devel:lib*) and cmd:gcc
```diff
-	lib:libgit2${secondaryArchSuffix}
-	lib:libyaml_cpp${secondaryArchSuffix}
+	lib:libgit2$secondaryArchSuffix
+	lib:libyaml_cpp$secondaryArchSuffix
```

Reply:
In cases where you can't use something like `devel:libfoo$secondaryArchSuffix` but need to revert to the provides in the package like `libfoo${secondaryArchSuffix}_devel`
For instance you could use `llvm17${secondaryArchSuffix}_devel`, but the package provides also `devel:libLLVM_17$secondaryArchSuffix` which is prefered.

Reply:
done